src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseChecklistService.cs

```diff
@@ -143,28 +149,31 @@ private async Task<bool> ReleaseHasNonEmptyHeadlineBlock(Guid releaseId)
 
         private async Task<bool> ReleaseHasEmptyGenericContentSection(Guid releaseId)
         {
-            return await _contentDbContext.ReleaseContentSections
-                .Include(rcs => rcs.ContentSection)
-                .ThenInclude(cs => cs.Content)
-                .Where(rcs =>
-                    rcs.ReleaseId == releaseId
-                    && rcs.ContentSection.Type == ContentSectionType.Generic)
-                .AnyAsync(rcs => rcs.ContentSection.Content.Count == 0);
+            return await _contentDbContext.ContentSections
+                .Where(cs =>
+                    cs.Type == ContentSectionType.Generic && 
+                    cs.Release.ReleaseId == releaseId)
+                .AnyAsync(cs => cs.Content.Count == 0);
         }
 
         private async Task<bool> ReleaseGenericContentSectionsContainEmptyContentBlock(Guid releaseId)
```

Comment:
I think it was me that originally named this - so I'm criticising myself here - but I'd be tempted to rename these methods to use `HtmlBlock` instead of `ContentBlock`. Same for the new related dashboards method.

Reply:
Replaced them with a single method `ReleaseSectionHasEmptyHtmlBlock`.